scripts/insert_into_appointments.sql

```diff
@@ -1 +1,15 @@
 -- insert appointments creating connections between doctors and patients
+
+-- -   Marsha Wilcox seeing Dr. Vicky Hilton at 10am on August 12, 2016.
+-- -   Ivory Talley seeing Dr. Spencer Hubbard at 1pm on August 17, 2016.
+-- -   Marsha Wilcox seeing Dr. Spencer Hubbard at 10am on August 12, 2016.
+-- -   Blake Hodge seeing Dr. Vicky Hilton at 1pm on August 17, 2016.
+
+
+INSERT INTO appointments(apt_date, apt_time, comments, doctor_id, patient_id)
+  SELECT TIMESTAMP '2016-08-16', 10:00, 'none', d.id, p.id
```

Comment:
`10:00` needs to be wrapped in single quotation marks.  This script only inserts one row into the table.

```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT
    DATE '2016-08-12',
    TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hilton' AND given_name = 'Vicky'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox' AND given_name = 'Marsha')
  UNION
  SELECT
    DATE '2016-08-17',
    TIME '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard' AND given_name = 'Spencer'),
    (SELECT id FROM patients WHERE family_name = 'Talley' AND given_name = 'Ivory')
  UNION
  SELECT
    DATE '2016-08-12',
    TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard' AND given_name = 'Spencer'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox' AND given_name = 'Marsha')
  UNION
  SELECT
    DATE '2016-08-17',
    TIME '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hilton' AND given_name = 'Vicky'),
    (SELECT id FROM patients WHERE family_name = 'Hodge' AND given_name = 'Blake')
```